### PR TITLE
Adds kubectl to toolbox

### PR DESCRIPTION
**Pull request by errm**
I need this for a script to bootstrap roles for k8s 1.6 rbac

### CODE REVIEW

**Comment by mtpereira** [comment]
LGTM! :shipit: :+1:

File: Dockerfile [hunk 1]
```diff
@@ -1,18 +1,23 @@
 FROM alpine:edge
 
+ENV K8S_VERSION=v1.6.3
```
Comment (Sodki) [inline Dockerfile]: Considering the goal of this image, should we just use latest instead? Do we care about the version at all, as long as it's recent?

Comment (errm) [reply Dockerfile]: tbh backwards compatibility has always been good with kubectl, I almost always run the latest version on my workstation and I have never had issues, so yes...

Comment (mtpereira) [reply Dockerfile]: We should keep in mind then that any `docker build` might change the `kubectl` version and therefor create a different image. Are we OK with this? It should be fine I guess, I just wanted to make sure that we're all on the same page.

Comment (Sodki) [reply Dockerfile]: I'm fine with that. And quay should build this periodically, to get shiny new version of things.

**Pull request merged**